jboss-integration/src/main/java/org/teiid/jboss/rest/RestASMBasedWebArchiveBuilder.java

```diff
@@ -117,7 +119,280 @@ else if (securityType.equalsIgnoreCase("httpbasic")) {
 		return byteStream.toByteArray();
 	}
 
-	private void writeEntry(String name, ZipOutputStream out, byte[] contents) throws IOException {
+    private void writeSwagger(ZipOutputStream out, Properties props, VDBMetaData vdb) throws IOException {
```

Comment:
This is not what I meant, I want you to add the "swagger" directory with its contents to the build somewhere, then just add the directory, that way when time to upgrade the swagger we can just replace that with new contents. We do NOT want to add one file by one file to the zip